apps/meteor/app/apps/server/bridges/livechat.ts

```diff
@@ -56,7 +57,7 @@ export class AppLivechatBridge extends LivechatBridge {
 		const appMessage = (await this.orch.getConverters().get('messages').convertAppMessage(message)) as IMessage | undefined;
 		const livechatMessage = appMessage as ILivechatMessage | undefined;
 
-		const msg = await LivechatTyped.sendMessage({
+		const msg = await sendMessage({
```

Comment:
should we call this function `sendMessage`? I'm thinking about a possible confusion with the "original" `sendMessage`, so I wonder if we should call this one something like `sendOmnichannelMessage` or maybe having an aggregate of all omnichannel functions so we'd still call them `Omnichannel.sendMessage`.. wdyt? Same applies to `createRoom` and so on